.github/workflows/upload-delegates.yml

```diff
@@ -0,0 +1,30 @@
+on: [pull_request]
+
+jobs:
+  upload_delegates:
+    runs-on: ubuntu-latest
+    name: Parses all the delegates and uploads all to IPFS
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Upload
+        id: upload
+        uses: makerdao-dux/github-action-deploy-delegates@v1.1.14
+        with:
+          delegates-folder: 'governance/delegates'
+          tags-file: 'governance/delegates/meta/tags.json'
+          infura-id: ${{ secrets.INFURA_ID }}
+          infura-secret: ${{ secrets.INFURA_KEY }}
+          output-file: 'governance/delegates/meta/delegates.json'
```

Comment:
To confirm, since this has a `output-file` specified, this will upload to IPFS _and_ output into the repo?

Reply:
yes!